app/javascript/app/pages/home/user-cases-section/user-cases-section-styles.scss

```diff
@@ -42,63 +55,75 @@
       justify-content: center;
       align-items: center;
       left: -35px;
-      top: -3px;
+      top: 0;
       color: $theme-color;
       font-weight: $font-weight-bold;
       border: 1px solid $theme-border;
       width: 26px;
       height: 26px;
       border-radius: 13px;
     }
-  }
-}
 
-.link {
-  color: $theme-color;
-  font-weight: $font-weight-bold;
-  position: relative;
-  display: inline-block;
-  text-shadow: 0 2px $white;
-
-  &::before {
-    content: "";
-    width: 100%;
-    height: 2px;
-    background-color: $theme-color;
-    position: absolute;
-    bottom: -1px;
-    left: 0;
-    z-index: -1;
+    a {
+      color: $theme-color;
+      font-weight: $font-weight-bold;
+      position: relative;
+      display: inline-block;
+      text-shadow: 0 2px $white;
+
+      &::before {
+        content: "";
+        width: 100%;
+        height: 2px;
+        background-color: $theme-color;
+        position: absolute;
+        bottom: 3px;
```

Comment:
I think you can set it to `1px` otherwise in some cases the underline overlaps with the text: 
![image](https://user-images.githubusercontent.com/6136899/50239781-f58d7680-03ba-11e9-975a-779937bd377c.png)

Reply:
@aabdaab I'm not being able to reproduce this issue (could it be some cache related thing?)
In which browser are you testing?